docs/source/faqs.md

```diff
@@ -76,7 +76,7 @@ ___
 
 **What is the average estimated savings for users??**
 
-This will vary but, in some cases, we are seeing 3x-10x savings. Typically, we offer 5-6x more price performance than hardware accelerators.
+We use CPUS and they are cheaper and more readily accessible than hardware accelerators. While savings will vary on the use case and business outcomes, we encourage you to benchmark our models to and measure cost savings for your specific use case.
```

Comment:
This whole section seems a bit ambiguous.  We don't really define what "savings" means, i.e. time, money, power, etc.

Reply:
Language is provided by Marketing. I can work with @NMsasa upon his return to refine it as needed.